README.md

```diff
@@ -68,6 +68,14 @@ Interested in contributing? Head over to our [contributing guide](CONTRIBUTING.m
 
 Contributions don't have to be code! They can be ideas, inspiration, discussion or filing bugs!
 
+### Developing
```

Comment:
@raveenakothapally Could you please move this into the CONTRIBUTING.md instead. Thanks.